### PR TITLE
Update install-machine.md to reference 0.3.1

### PR DESCRIPTION
**Pull request by jgeiger**
Update the docs to include links to newest release.

### CODE REVIEW

**Comment by nathanleclaire** [comment]
Ah, awesome, but you need to sign your commit

**Comment by jgeiger** [comment]
Not sure if I need to mention it, but I did sign the commit.

File: docs/install-machine.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ weight=3
 Docker Machine is supported on Windows, OS X, and Linux and is installable as
 one standalone binary.  The links to the binaries for the various platforms and
 architectures are available at the [Github
-Release](https://github.com/docker/machine/releases/tag/v0.3.0) page.
+Release](https://github.com/docker/machine/releases/tag/v0.3.1) page.
```
Comment (thaJeztah) [inline docs/install-machine.md]: I think this link could be changed to https://github.com/docker/machine/releases, which shows an overview of all releases - one link less to update :-)

**Comment by thaJeztah** [comment]
LGTM, thanks @jgeiger

**Comment by jgeiger** [comment]
Removed the specific version from the binaries link per @thaJeztah. I think this make sense since it's not a command you copy/paste.

**Comment by thaJeztah** [comment]
Thanks @jgeiger !

ping @nathanleclaire for the merge (unless you need another "official" LGTM) :smile:

**Comment by nathanleclaire** [comment]
LGTM

**Pull request merged**